src/test/java/core/basesyntax/serviceimpl/ReaderServiceImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.serviceimpl;

```

Comment:
```suggestion
package core.basesyntax.service.impl;

```
OR
```suggestion
package core.basesyntax.service;

```
same for other service test classes